internal/builders/maven/README.md

```diff
@@ -86,25 +88,32 @@ jobs:
 
 Now, when you invoke this workflow, the Maven builder will build both your artifacts and the provenance files for them.
 
-You can also release artifacts to Maven Central by adding the following step to your workflow:
-
-```yaml
-  publish:
-    needs: build
-    uses: slsa-framework/slsa-github-generator/.github/workflows/publish_maven.yml@v1.7.0
-    with:
-      provenance-download-name: "${{ needs.build.outputs.provenance-download-name }}"
-      provenance-download-sha256: "${{ needs.build.outputs.provenance-download-sha256 }}"
-      target-download-sha256: "${{ needs.build.outputs.target-download-sha256 }}"
-    secrets:
-      maven-username: ${{ secrets.OSSRH_USERNAME }}
-      maven-password: ${{ secrets.OSSRH_PASSWORD }}
-      gpg-key-pass: ${{ secrets.GPG_PASSPHRASE }}
-      gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+### Releasing to Maven Central
+
+You can also release artifacts to Maven Central with [the slsa-github-generator Maven publish action](https://github.com/slsa-framework/slsa-github-generator/blob/main/actions/maven/publish/README.md).
+
+#### Action requirements
+
+Besides adding the above workflow to your CI pipeline, you also need to add the following plugin to your `pom.xml`:
+
+```xml
+<plugin>
+    <groupId>dev.slsa.slsaframework</groupId>
```

Comment:
in the slsa-verifier, we use `io.github.slsa-framework`. Can we use the same for consistency?

Reply:
Let's go with `io.github.slsa-framework.slsa-github-generator`. I used `io.github.slsa-framework.slsa-verifier` for the verifier plugin in https://github.com/slsa-framework/slsa-verifier/pull/676